mobile/ios/Runner.xcodeproj/project.pbxproj

```diff
@@ -796,9 +805,11 @@
 				ASSETCATALOG_COMPILER_APPICON_NAME = "AppIcon-test";
 				CLANG_ENABLE_MODULES = YES;
 				CODE_SIGN_IDENTITY = "Apple Development";
-				CODE_SIGN_STYLE = Automatic;
-				CURRENT_PROJECT_VERSION = "$(FLUTTER_BUILD_NUMBER)";
-				DEVELOPMENT_TEAM = V3D64P9D6W;
+				"CODE_SIGN_IDENTITY[sdk=iphoneos*]" = "iPhone Distribution";
+				CODE_SIGN_STYLE = Manual;
+				CURRENT_PROJECT_VERSION = 1805;
```

Comment:
🤔 can't we use the $(FLUTTER_BUILD_NUMBER) anymore? it seems to be a bit of nuisance to update this one manually in multiple places - it will be forgotten sooner or later 🙁

in the worst case we could do this manually with a `just set-ipa-version $VERSION` command that just runs `sed` under the hood

Reply:
I believe it is autogenerated now and not controlled via Flutter anymore, i.e. it will be overwritten everytime you build it. Hence, the change in this file. But maybe I just change it back since we don't check in the build changes after the CI run.

Reply:
does this mean that our build numbering will change?

will the version seen inside the app match the new one?

Reply:
We are still using dynamic build number which are generated the same way as before just in a different place: 

https://github.com/get10101/10101/blob/09e2b63a99104ab76bd311940d67494a2e121855/mobile/ios/fastlane/Fastfile#L49-L53